scripts/nettest/simpleserver

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+set -e
+
+while true
+do
+    echo -e "HTTP/1.1 200 OK\r\n\r\nHello World" | nc -l -p 8080
```

Comment:
This will peg a thread at 100%, right? Is that okay, or should we use some small delay in the loop or something?

Reply:
I think we should go ahead and merge this to un-break rootless E2E, but still wondering if this should be edited.

Reply:
I tried it locally and don't see it taking much CPU. I think it just waits on the port.

Reply:
`nc` will wait for a connection which won't take much CPU resources.
Once it gets a connection it responds and exits, so the loop just makes sure it is re-run again (and listens again)

Reply:
Nice!